clifford/test/test_clifford.py

```diff
@@ -196,6 +196,54 @@ def test_add_float64(self, g3):
 
         assert 1 + e1 == e1 + np.float64(1)
 
+    def _random_value_array(self, layout, Nrows, Ncolumns):
+        value_array = np.zeros((Nrows, Ncolumns, layout.gaDims))
+        for i in range(Nrows):
+            for j in range(Ncolumns):
+                value_array[i, j, :] = layout.randomMV().value
+        return value_array
+
+    def test_2d_mv_array(self, g3):
+        layout, blades = g3, g3.blades
+        Nrows = 2
+        Ncolumns = 3
+        value_array_a = self._random_value_array(g3, Nrows, Ncolumns)
+        value_array_b = self._random_value_array(g3, Nrows, Ncolumns)
+
+        mv_array_a = MVArray.from_value_array(layout, value_array_a)
+        assert mv_array_a.shape == (Nrows, Ncolumns)
+        mv_array_b = MVArray.from_value_array(layout, value_array_b)
+        assert mv_array_b.shape == (Nrows, Ncolumns)
+
```

Comment:
```suggestion
# check the array is preserved (no need to check both)
assert_array_equal(mv_array_a.value, value_array_a)
assert mv_array_a.value.dtype == value_array_a
assert type(mv_array_a.value) == type(value_array_a)

```

Reply:
This produces the error
```
        # check properties of the array are preserved (no need to check both a and b)
        np.testing.assert_array_equal(mv_array_a.value, value_array_a)
>       assert mv_array_a.value.dtype == value_array_a
E       TypeError: data type not understood

test_clifford.py:249: TypeError
```
With or without the change above to add .view(ndarray)

Reply:
I missed a `.dtype`, updated.